examples/Shop/ShopPage.php

```diff
@@ -50,10 +50,6 @@ function generateHeader()
 
 function generateBody()
 {
-	$pDateTime = new DateTime();
```

Comment:
This is a relict from when the provider could define the abo start date. So this can definitely be removed.